.github/copilot-instructions.md

```
@@ -0,0 +1,74 @@
+- Use a `config_flow.py` file for configuration flows.
+
+## Coding Standards
+- Add typing annotations to all functions and classes, including return types.
+- Add descriptive docstrings to all functions and classes (PEP 257 convention). Update existing docstrings if needed.
+- Keep all existing comments in files.
+- Ensure all imports are put at the top of the file.
+- Pre-commit hooks are configured in `/.pre-commit-config.yaml`.
+- Ruff enforces code style (settings in `/pyproject.toml`).
+- mypy enforces static typing (settings in `/pyproject.toml`).
+
+## Local tooling note
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)